packages/@aws-cdk/aws-servicecatalogappregistry/lib/application-associator.ts

```diff
@@ -4,9 +4,21 @@ import { IApplication, Application } from './application';
 import { CheckedStageStackAssociator } from './aspects/stack-associator';
 
 /**
- * Properties for a Service Catalog AppRegistry AutoApplication
+ * Properties for Service Catalog AppRegistry Application Associator
  */
 export interface ApplicationAssociatorProps {
+  /**
+   * Application associator properties.
+   *
+   * @default - Empty array.
+   */
+  readonly applications: IBaseApplicationAssociatorProps[];
```

Comment:
```suggestion
  readonly applications: ITargetApplication[];
```

* Props never start with `I`
* `BaseXxxProps` are called `XxxOptions`
* These aren't props for the `ApplicationAssociator`, they relate to `targetApplications` (or whatever)